specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/billingV2.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-03-01-preview",
```

Comment:
Why add this v2 to `2018-03-01-preview`?

Reply:
The older version of PowerShell cmdlets are using the these API. We were planning to deprecate these but the next PowerShell breaking change release is after 3 months so we are blocked. SDK team suggested to introduce the new json file with limited number of APIs so that we can use it to generate the SDKs.